scrapers/AssumeThePositionStudios.py

```diff
@@ -0,0 +1,123 @@
+import json
+import sys
+import requests
+import re
+import os
+
+try:
+    import py_common.graphql as graphql
+    import py_common.log as log
+except ModuleNotFoundError:
+    print(
+        "You need to download the folder 'py_common' from the community repo! (CommunityScrapers/tree/master/scrapers/py_common)",
+        file=sys.stderr,
+    )
+    sys.exit()
+
+
+def debugPrint(t):
+    #sys.stderr.write(t + "\n");
```

Comment:
This could just delegate to `log.debug`, it's very helpful when scrapers actually log something so we can find errors faster 🙂

You could also just replace all calls of `debugPrint` with `log.debug`, and adding a trailing newline is not necessary since the log module already takes care of flushing the output

Reply:
This needs to be fixed either way because it's a syntax error and this script won't run at all in its current state 😅